enterprise/server/scheduling/task_router/task_router.go

```diff
@@ -124,35 +124,41 @@ func (tr *taskRouter) RankNodes(ctx context.Context, cmd *repb.Command, remoteIn
 		return nonePreferred(nodes)
 	}
 
-	nodeByID := map[string]interfaces.ExecutionNode{}
+	nodeByExecutorID := map[string]interfaces.ExecutionNode{}
```

Comment:
Do we need this?

Reply:
Looks like this is still here, maybe forgot to push?